tools/test_api.py

```diff
@@ -2122,7 +2128,6 @@ def build_tests(tests, base_source_paths, build_path, target, toolchain_name,
 
     Returns a tuple of the build result (True or False) followed by the test
     build data structure"""
-
```

Comment:
I think I just missed that change when I had some debug prints. If you feel REALLLY strongly about it I'll remove it, otherwise I might just let it simmer :)